dashboard/app/controllers/lti/v1/account_linking_controller.rb

```diff
@@ -18,8 +18,8 @@ def finish_link
         # clear the session, so we need to store cache key first, sign them out,
         # then set it again in their now empty session.
         if current_user
-          partial_registration_cache_key = session[PartialRegistration::SESSION_KEY]
-          sign_out current_user
+          partial_registration_cache_key = PartialRegistration.cache_key(current_user)
```

Comment:
Was this change accidentally created by branching off a slightly outdated staging? Changing to pulling the key from the session was part of [this fix](https://github.com/code-dot-org/code-dot-org/pull/59368) for Schoology logouts

Reply:
Yep that's a rebase issue, fixed this